static/src/javascripts/projects/common/modules/email/run-checks.js

```diff
@@ -194,6 +204,7 @@ define([
         getEmailInserted: getEmailInserted,
         allEmailCanRun: allEmailCanRun,
         getUserEmailSubscriptions: getUserEmailSubscriptions,
-        listCanRun: listCanRun
+        listCanRun: listCanRun,
+        checkTestClash: checkTestClash
```

Comment:
Yeah, using internally is fine, but if we're only exporting it for a unit test, differentiating is a good idea :)